node-js-basics-101/node_modules/debug/src/inspector-log.js

```diff
@@ -0,0 +1,15 @@
+module.exports = inspectorLog;
+
+// black hole
+const nullStream = new (require('stream').Writable)();
+nullStream._write = () => {};
+
+/**
+ * Outputs a `console.log()` to the Node.js Inspector console *only*.
+ */
+function inspectorLog() {
+  const stdout = console._stdout;
```

Comment:
__[jshint]__
* 'const' is available in ES6 (use 'esversion: 6') or Mozilla JS extensions (use moz).

<!-- Comment by [SideCI](https://sideci.com) -->